app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from 'react-router-dom';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state);
+
+  function getGreetings () {
+    console.log('getThings() Action!!');
+    return dispatch => {
+      dispatch({ type: 'GET_THINGS_REQUEST' });
+      return fetch('v1/things.json')
+      .then(response => response.json())
+      .then(json => dispatch(getThingsSuccess(json)))
+      .then(error => console.log(error));
```

Comment:
- [ ] Kindly remove console log statements. It is ugly and exposes your data in the browser console.